community/modules/scheduler/htcondor-configure/files/htcondor_configure.yml

```diff
@@ -149,6 +152,56 @@
           -token condor@{{ trust_domain }}
       args:
         creates: "{{ condor_config_root }}/tokens.d/condor@{{ trust_domain }}"
+    - name: Enable SchedD high availability
+      when: job_queue_ha | bool
+      block:
+      - name: Setup HA Spool directory
+        ansible.builtin.file:
+          path: "{{ spool_dir }}"
+          state: directory
+          owner: condor
+          group: condor
+          mode: 0700
+      - name: Set SchedD HA configuration (requires restart)
+        ansible.builtin.copy:
+          dest: "{{ condor_config_root }}/config.d/{{ schedd_ha_config_file }}"
+          mode: 0644
+          content: |
+            MASTER_HA_LIST=SCHEDD
+            HA_LOCK_URL=file:{{ spool_dir }}
+            VALID_SPOOL_FILES=$(VALID_SPOOL_FILES), SCHEDD.lock
+            HA_POLL_PERIOD=30
+        notify:
+        - Restart HTCondor
+      - name: Create SystemD override directory for HTCondor
```

Comment:
FYI: this SystemD override can be removed once HTCondor releases the fixes. it is harmless once they do.

TL;DR: it tells SystemD to start HTCondor only after any fstab entries necessary to access that the `{{ spool_dir }}` have been mounted.

Reply:
I would add a `# TODO` in the code. If you disagree with that then perhaps a task scheduled for future. I am just betting we will not come back to this comment.

Reply:
I added a comment in the code explaining when it can be removed and also created a task. It is front and center in my mind but it is good to leave a trail of crumbs.